scripts/owl-rules-check.py

```
@@ -48,7 +48,11 @@ def closure(*paths):
     """Parse the given Turtle files into one graph and expand its OWL-RL closure."""
     g = rdflib.Graph()
     for p in paths:
-        g.parse(p, format="turtle")
+        try:
+            g.parse(p, format="turtle")
+        except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)